development/tools/pkg/githubrelease/storage.go

```diff
@@ -8,19 +8,38 @@ import (
 	"github.com/kyma-project/test-infra/development/tools/pkg/common"
 )
 
-// StorageAPIWrapper abstracts Google Storage API
-type StorageAPIWrapper struct {
-	Context       context.Context
-	StorageClient *storage.Client
-	BucketName    string
-	FolderName    string
+// StorageAPI exposes a function to read objects from Google Storage buckets
+type StorageAPI interface {
+	ReadBucketObject(ctx context.Context, fileName string) ([]byte, error)
+}
+
+// storageAPIWrapper implements reading objects from Google Storage buckets
+type storageAPIWrapper struct {
+	storageClient *storage.Client
+	bucketName    string
+	folderName    string
+}
+
+// NewStorageAPIWrapper returns new instance of storageAPIWrapper
+func NewStorageAPIWrapper(ctx context.Context, bucketName, folderName string) (StorageAPI, error) {
```

Comment:
Same question as for GithubAPI: is Wrapper necessary?

Reply:
done